src/components/fields/TraceSelector.js

```diff
@@ -64,17 +68,30 @@ class TraceSelector extends Component {
     } else {
       this.traceOptions = [{label: 'Scatter', value: 'scatter'}];
     }
+    this.fullValue = plotlyTraceToCustomTrace(props.container);
+  }
 
-    this.fullValue = plotlyTraceToCustomTrace(props.fullContainer);
+  setTraceDefaults(container, fullContainer, updateContainer) {
+    if (
+      (container.type === 'scatter' && !container.mode) ||
+      (!container.type && fullContainer.type === 'scatter')
+    ) {
+      updateContainer({
+        type: 'scatter',
+        mode: fullContainer.mode || 'markers',
```

Comment:
I don't have a real strong opinion here, but I probably would have gone for `'lines+markers'` as that's the "<20 points" default and we get here only (I think?) when there are no points.

Reply:
I put markers, because it's the editor default..
We're using plotly.js defaults wherever we can with `fullContainer.mode`, but when we're at our first trace that is empty and didn't have any data put in yet, we put in the editor default of 'markers'